build/source/netcdf/def_output.f90

```diff
@@ -469,14 +469,14 @@ subroutine write_hru_info(ncid, err, message)
  do iGRU = 1, size(gru_struc)
 
   ! GRU info
-  err = nf90_put_var(ncid, gruVarID, gru_struc(iGRU)%gru_nc, start=(/iGRU/))
+  err = nf90_put_var(ncid, gruVarID, gru_struc(iGRU)%gruId, start=(/iGRU/))
```

Comment:
Is it really gruID and hru_id? That seems somewhat inconsistent (but a separate issue as long as this is correct)

Reply:
Yep :(